frontend/src/services/user/types.ts

```diff
@@ -55,4 +55,12 @@ export type UserAccountLoader = {
 export type PublicUserInfo = {
   username: string;
   avatar: string;
+  stats: {
+    purchases: number;
+    sales: number;
+    sales_this_month: number;
+    seller_rating: number | string;
```

Comment:
Why can this be either a number or a string?

Reply:
it comes from the backend as string because i am formating it there and sending it but I kept the number if I need to change it. If this is no ok, I can change it to number and then format it in the frontend ?